tests/Tests/600__ai_apps/600__anaconda_commercial_edition/600__anaconda_commercial_edition.robot

```diff
@@ -15,9 +25,89 @@ Verify Anaconda Commercial Edition Is Available In RHODS Dashboard Explore Page
   Verify Service Provides "Get Started" Button In The Explore Page    Anaconda Commercial Edition
   Verify Service Provides "Enable" Button In The Explore Page    Anaconda Commercial Edition
 
-* Keywords ***
+Verify Anaconda Commercial Edition Fails Activation When Key Is Invalid
+  [Tags]  Sanity
+  ...     ODS-310  ODS-367
+  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  Wait for RHODS Dashboard to Load
+  Enable Anaconda  ${invalid_key}
+  Wait Until Keyword Succeeds    30  1  Check Connect Button Status  false
+  Capture Page Screenshot  anaconda_failed_activation.png
+  ${text} =  Get Text  xpath://*[@class="pf-c-form__alert"]
+  Should Be Equal  ${text}  ${error_msg}
+  Click Button    Cancel
+  Menu.Navigate To Page    Applications    Enabled
+  ## Page Should Not Contain  Anaconda Commercial Edition
+  Wait Until RHODS Dashboard JupyterHub Is Visible
+  Capture Page Screenshot  enabletab_anaconda_notpresent.png
+  Page Should Not Contain Element  xpath://div[@class="pf-c-card__title"]/span[.="Anaconda Commercial Edition"]
+
+Verify User Is Able to Activate Anaconda Commercial Edition
+  [Tags]  Sanity
```

Comment:
How about using this in Smoke?

Reply:
well, given that the Smoke checks if the Anaconda tile is present in the explorer page, I think it makes sense to include the check on activation too. I would set it as both Sanity and Smoke because of the issue RHODS-2186. What do you think?